### DISCUSSION TITLE
mail for kovidgoyal

### DISCUSSION THREAD

**Discussion by j0k4rX** [kovidgoyal/kitty#5558] [General]
Hi,
what (IDEs or tools) do u use for developing linux apps? i'm curious.
i searched a lot, but wasn't enough.
Best.

**Comment by kovidgoyal** [comment]
On Wed, Oct 05, 2022 at 08:43:49AM -0700, j0k4rX wrote:
> Hi,
> what (IDEs or tools) do u use for developing linux apps? i'm curious.
> i searched a lot, but wasn't enough.
> Best.

vim and a whole bunch of tools I have written for myself over the years.

**Reply by DeveloperKev** [reply]
> On Wed, Oct 05, 2022 at 08:43:49AM -0700, j0k4rX wrote: Hi, what (IDEs or tools) do u use for developing linux apps? i'm curious. i searched a lot, but wasn't enough. Best.
> vim and a whole bunch of tools I have written for myself over the years.

shareable? 
i.e. tools like cmake builds/scripts?

**Comment by kovidgoyal** [comment]
On Tue, Oct 11, 2022 at 05:05:34AM -0700, ™ wrote:
> > On Wed, Oct 05, 2022 at 08:43:49AM -0700, j0k4rX wrote: Hi, what (IDEs or tools) do u use for developing linux apps? i'm curious. i searched a lot, but wasn't enough. Best.
> > vim and a whole bunch of tools I have written for myself over the years.
> 
> shareable? 
> i.e. tools like cmake builds/scripts?

Build systems, for example https://github.com/kovidgoyal/bypy 
a browser:  https://github.com/kovidgoyal/vise
various kittens in kitty: for example the diff, unicode input and ssh ones.

and lots of custom scripts that setup my whole environment, all the way
from session management and machine provision on down. These have been
written over many years and they are very specific to my workflow so not
really shareable.

**Comment by DeveloperKev** [comment]
is visual studio ideal for linux/crossplatform development? what do u suggest?
which os r u in btw?

**Comment by kovidgoyal** [comment]
never used visual studio so no idea. I use linux.

On Tue, Oct 11, 2022 at 06:01:41AM -0700, ™ wrote:
> is visual studio ideal for linux as well as crossplatform app development? what do u suggest?
> which os r u in btw?
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/kovidgoyal/kitty/discussions/5558#discussioncomment-3850364
> You are receiving this because you commented.
> 
> Message ID: ***@***.***>

-- 
_____________________________________

Dr. Kovid Goyal 
https://www.kovidgoyal.net
https://calibre-ebook.com
_____________________________________
